pkg/bucketeer/event/processor.go

```diff
@@ -28,13 +28,13 @@ import (
 // every time the specified time elapses or the specified capacity is exceeded.
 type Processor interface {
 	// PushEvaluationEvent pushes the evaluation event to the queue.
-	PushEvaluationEvent(ctx context.Context, user *protouser.User, evaluation *protofeature.Evaluation)
+	PushEvaluationEvent(ctx context.Context, user *protouser.User, evaluation *protofeature.Evaluation, tag string)
```

Comment:
Since the tag doesn't change and now it is being passed in all push interfaces, maybe it would be better to pass the tag when the processor is initialized and simplify the interface. What do you think?

Reply:
> maybe it would be better to pass the tag when the processor is initialized and simplify the interface. What do you think?

I agree 👍

Reply:
Done. Please check it.